engine/src/main/resources/templates/Rbac/customGroupCreatedEmailBody.html

```diff
@@ -0,0 +1,52 @@
+{#include Rbac/insightsEmailBody}
+{#content-title}Rbac group change Notification - {action.timestamp.toStringFormat()}{/content-title}
+{#content-body}
+<tr>
+    <td class="rh-body rh-multi-column">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td>
+                    <!-- end 50 col -->
+                    <!--[if (gte mso 9)|(IE)]>
+                    </td>
+                    </tr>
+                    </table>
+                    <![endif]-->
+                </td>
+            </tr>
+            <!-- end content block -->
+            </tbody>
+        </table>
+    </td>
+</tr>
+<!-- end body section -->
+
+<!-- Body section -->
+<tr>
+    <td class="rh-body">
+        <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0" cellspacing="0" width="100%">
+            <tbody>
+            <!-- Content block -->
+            <tr>
+                <td class="rh-content__block">
+                    <p>
+                        Hi,
+                        <br>
+                        <br>
+                        A custom group has been created by {action.events[0].payload.username}.
+                        <br>
+                        Check the created group <a class="rh-url" href="{environment.url('/settings/rbac/groups/detail/' + action.events[0].payload.uuid)}"><b>{action.events[0].payload.name}</b></a>
```

Comment:
Unfortunately, `'/settings/rbac/groups/detail/' + action.events[0].payload.uuid` will be parsed as a string and the output will look like this:
```
https://console.stage.redhat.com/settings/rbac/groups/detail/'%20+%20action.events%5B0%5D.payload.uuid
```
Here's how to use an environment-based dynamic URL and append something at its end:
```suggestion
                        Check the created group <a class="rh-url" href="{environment.url('/settings/rbac/groups/detail/')}{action.events[0].payload.uuid}"><b>{action.events[0].payload.name}</b></a>
```

Reply:
Good to know, thanks!